src/handlers/assign.rs

```diff
@@ -41,8 +41,13 @@ mod tests {
 const NEW_USER_WELCOME_MESSAGE: &str = "Thanks for the pull request, and welcome! \
 The Rust team is excited to review your changes, and you should hear from {who} soon.";
 
-const CONTRIBUTION_MESSAGE: &str = "\
-Please see [the contribution instructions]({contributing_url}) for more information.";
+const CONTRIBUTION_MESSAGE: &str = "Please see [the contribution
+instructions]({contributing_url}) for more information. Namely, in order to ensure the
```

Comment:
Hm ... ok, so @ehuss how should the text be formatted? IIUC, you're suggesting the following formatting:

```rust
const CONTRIBUTION_MESSAGE: &str = "Please see [the contribution
instructions]({contributing_url}) for more information. Namely, in order to ensure the
minimum review times lag, PR authors and assigned reviewers should ensure that the review
label (`S-waiting-on-review` and `S-waiting-on-author`) stays updated, invoking these commands 
when appropriate:

`@rustbot author`: the review is finished, PR author should check the comments and take action accordingly
`@rustbot review`: the author is ready for a review, this PR will be queued again in the reviewer's queue";
```

correct?

Reply:
I think it would also need backslashes in the first paragraph (so the first 4 words aren't on a line by themselves). Maybe something like:

```rust
const CONTRIBUTION_MESSAGE: &str = "Please see [the contribution \
instructions]({contributing_url}) for more information. Namely, in order to ensure the \
minimum review times lag, PR authors and assigned reviewers should ensure that the review \
label (`S-waiting-on-review` and `S-waiting-on-author`) stays updated, invoking these commands \
when appropriate:

`@rustbot author`: the review is finished, PR author should check the comments and take action accordingly
`@rustbot review`: the author is ready for a review, this PR will be queued again in the reviewer's queue";
```

Reply:
We have a new PR with the new wording  (108630, no link to avoid noise) and now we also see how it looks like.

I'm going to submit a small additional patch to improve the formatting. Thanks @ehuss and sorry about that!